### PR TITLE
MISUV-3594: display MFA credits

### PR DESCRIPTION
**Pull request by avelx**

### CODE REVIEW

File: app/views/CreditAndRefunds.scala.html [hunk 1]
```diff
@@ -103,13 +104,34 @@ <h2 class="govuk-heading-m">@{balance.get.firstPendingAmountRequested.get.toCurr
...
-                                <a href="@getControllerHref(charge._1.documentDetail, isAgent)" class="govuk-link">@charge._1.dueDate.get.toLongDate @messages("credit-and-refund.payment")</a>
-                            </dt>
-                            <dd class="govuk-summary-list__value">@charge._1.documentDetail.paymentOrChargeCredit.get.toCurrencyString</dd>
-                        </div>
+                    @for(charge <- creditCharges){
+                        @if(isMFACreditsAndDebitsEnabled){
+                            @if(charge._1.documentDetail.validMFACreditDescription()) {
+                                <div class="govuk-summary-list__row">
+                                    <dt class="govuk-summary-list__value">
+                                        @* => MISUV-3648*@
```
Comment (Joshtest1570) [inline app/views/CreditAndRefunds.scala.html:112]: Please remove the ticket number from the code.

File: conf/messages [hunk 2]
```diff
@@ -820,6 +820,7 @@ credit-and-refund.details-intro                                   = What’s in
 credit-and-refund.no-credit                                       = You have no money in your account.
 credit-and-refund.claim-refund-btn                                = Claim a refund
 credit-and-refund.check-refund-btn                                = Check refund status
+credit-and-refund.credit-from-hmrc-title                          = Credit from HMRC adjustment
```
Comment (Joshtest1570) [inline conf/messages:823]: Please add Welsh translation as well.

File: test/controllers/CreditAndRefundControllerSpec.scala [hunk 3]
```diff
@@ -87,10 +108,13 @@ class CreditAndRefundControllerSpec extends MockAuthenticationPredicate with Moc
 
       "redirect to the custom not found error page" in new Setup {
         disable(CreditsRefundsRepay)
+        // redirect even in case MFACreditsAndDebits is ON
```
Comment (Joshtest1570) [inline test/controllers/CreditAndRefundControllerSpec.scala:111]: Please can you remove this comments

File: conf/messages (hunk 2 above)
Comment (avelx) [reply conf/messages:823]: welsh translation added

File: test/controllers/CreditAndRefundControllerSpec.scala (hunk 3 above)
Comment (avelx) [reply test/controllers/CreditAndRefundControllerSpec.scala]: comment removed

File: app/views/CreditAndRefunds.scala.html (hunk 1 above)
Comment (avelx) [reply app/views/CreditAndRefunds.scala.html]: comment removed

**Review by Joshtest1570 — APPROVED** [review]

**Pull request merged**